rotkehlchen/types.py

```diff
@@ -838,3 +838,11 @@ class HistoryEventQueryType(SerializableEnumNameMixin):
     ETH_WITHDRAWALS = auto()
     BLOCK_PRODUCTIONS = auto()
     EXCHANGES = auto()
+
+
+class CacheInvalidationType(SerializableEnumNameMixin):
+    ICONS = auto()
+    AVATARS = auto()
+    CURVE = auto()
+    YEARN = auto()
+    MAKERDAO = auto()
```

Comment:
As I said kiss for me here is: query everything but if you understood differently coordinate in discord